trieste/acquisition/function.py

```diff
@@ -163,9 +163,20 @@ def prepare_acquisition_function(
         self, dataset: Dataset, model: ModelInterface
     ) -> AcquisitionFunction:
         """
-        Need to sample possible min-values from our posterior. 
+        Need to sample possible min-values y* from our posterior. 
         To do this we implement a Gumbel sampler.
-        We approximate Pr(y*^hat<y) by Gumbel(a,b) then sample from Gumbel.
+        We approximate Pr(y*<y) by Gumbel(a,b) then sample from Gumbel.
+
+        The Gumbel distribution for minimas has a cumulative density function
+        of f(y)= 1 - exp(-1 * exp((y - a) / b)), i.e. the q^th quantile is given by
```

Comment:
```suggestion
        of :math:`f(y)= 1 - exp(-1 * exp((y - a) / b))`, i.e. the q\ :sup:`th` quantile is given by
```
and could you add the `:math:` context elsewhere please?

Reply:
also, was the `-1 * exp(...)` intentional over `- exp(...)`?

Reply:
@awav why resolved?